main.js

```diff
@@ -102,28 +102,28 @@ layers.hills.on("postrender", function (evt) {
 })
 
 const destinations = [
-  [[-11509256.048274003, -4792688.738366079], "Shanklin"],
```

Comment:
Jamie B suggested keeping the old ones, and I think that's a good idea!

Can you change this so that we add the new name and keep the old, e.g. for Shanklin we'd end up with:
```js
  [[-11509256.048274003, -4792688.738366079], "Shanklin"],
  [[-11509256.048274003, -4792688.738366079], "Gampos"],
```